tools/expotools/src/publish-packages/tasks/updateIosProjects.ts

```diff
@@ -39,13 +39,9 @@ export const updateIosProjects = new Task<TaskArgs>(
           return;
         }
 
-        logger.log(
-          '  ',
-          `${green(nativeApp.packageName)}: updating`,
-          podspecNames.map((podspecName) => green(podspecName!)).join(', ')
-        );
+        logger.log('  ', `${green(nativeApp.packageName)}: Reinstalling pods...`);
 
-        await spawnAsync('pod', ['update', ...podspecNames, '--no-repo-update'], {
+        await spawnAsync('pod', ['install', '--no-repo-update'], {
```

Comment:
What we could do to make it even more fool-proof would be to update iOS projects **before** publishing/making any changes — if the command would make any changes in the directories that would mean that the state is not clean and in order to make the _Publish packages_ commit clean and single-purposed one would need to first update Pods directories by reinstalling pods and only afterwards return to publishing packages.

Reply:
Interesting idea 🤔 I'm only afraid that installing pods is the bottleneck in terms of the performance, it takes the most time of the entire command, so running it twice would be quite annoying 😞